bin/ci_glob_setup.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+#######################################
+# (c) Copyright IBM Corp. 2021
+# (c) Copyright Instana Inc. and contributors 2021
+#######################################
+
+
+# Each of these environment variables will hold a list of files per CircleCI process to be tested.
+# They will be used later by each package where the `test:mocha` script is present
+# The `circleci tests glob` command list the files based on the provided glob. A similar result can be achieved without
+# their tool, but CircleCI recommends its usage.
+# The `circleci tests split` command will split files among the number of designated processes.
+# The `--split-by=timings` option looks at the previous build and downloads all result*.xml files. Then, it attempts to
+# split files among the process in a way that they finish more or less at the same time.
+
+export CI_AUTOPROFILE_TEST_FILES=$(circleci tests glob "packages/autoprofile/test/**/*test.js" | circleci tests split --split-by=timings | sed 's/packages\/autoprofile\///g') \
+export CI_OTEL_EXPORTER_TEST_FILES=$(circleci tests glob "packages/opentelemetry-exporter/test/**/*test.js" | circleci tests split --split-by=timings | sed 's/packages\/opentelemetry-exporter\///g') \
+export CI_AWS_FARGATE_TEST_FILES=$(circleci tests glob "packages/aws-fargate/test/**/*test.js" | circleci tests split --split-by=timings | sed 's/packages\/aws-fargate\///g') \
+export CI_COLLECTOR_TEST_FILES=$(circleci tests glob "packages/collector/test/**/*test.js" | circleci tests split --split-by=timings | sed 's/packages\/collector\///g') \
+export CI_AWS_LAMBDA_TEST_FILES=$(circleci tests glob "packages/aws-lambda/test/**/*test.js" | circleci tests split --split-by=timings | sed 's/packages\/aws-lambda\///g') \
+export CI_CORE_TEST_FILES=$(circleci tests glob "packages/core/test/**/*test.js" | circleci tests split --split-by=timings | sed 's/packages\/core\///g') \
+export CI_GOOGLE_CLOUD_RUN_TEST_FILES=$(circleci tests glob "packages/google-cloud-run/test/**/*test.js" | circleci tests split --split-by=timings | sed 's/packages\/google-cloud-run\///g') \
+export CI_LEGACY_SENSOR_TEST_FILES=$(circleci tests glob "packages/legacy-sensor/test/**/*test.js" | circleci tests split --split-by=timings | sed 's/packages\/legacy-sensor\///g') \
+export CI_METRICS_UTIL_TEST_FILES=$(circleci tests glob "packages/metrics-util/test/**/*test.js" | circleci tests split --split-by=timings | sed 's/packages\/metrics-util\///g') \
+export CI_SERVERLESS_TEST_FILES=$(circleci tests glob "packages/serverless/test/**/*test.js" | circleci tests split --split-by=timings | sed 's/packages\/serverless\///g') \
+export CI_SHARED_METRICS_TEST_FILES=$(circleci tests glob "packages/shared-metrics/test/**/*test.js" | circleci tests split --split-by=timings | sed 's/packages\/shared-metrics\///g')
```

Comment:
Would it be worth creating these env vars in a dynamic fashion? It would work automatically when we add more packages later.

Something like this could work (not tested completely, since the circleci command is not available locally):


```
set -xeo pipefail

cd `dirname $BASH_SOURCE`/..

for package in packages/*/ ; do
  echo $package
  # remove leading "packages/"
  package_name=${package#packages/}
  # remove trailing "/"
  package_name=${package_name%/}
  # replace all "-" by "_"
  package_name=${package_name//-/_}
  # convert lower case to upper case
  package_name="${package_name^^}"

  declare CI_${package_name}_TEST_FILES=$(circleci tests glob "${package}test/**/*test.js" | circleci tests split --split-by=timings | sed "s|${package}||g" ) 
  export CI_${package_name}_TEST_FILES
done
```